rqt_image_view/src/rqt_image_view/ratio_layouted_frame.cpp

```diff
@@ -30,6 +30,7 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
+#include <ros/ros.h>
```

Comment:
Why did you add this include? It doesn't seem necessary for any of the other changes. If that is the case please remove the line.